README-CN.md

```diff
@@ -0,0 +1 @@
+# 自由衫
```

Comment:
Is this literally just "README" in chinese?

If so, great initiative. This will increase this projects demographic scope.

Reply:
No its Free Tshirts to match our current readme

I feel this phrase will really resinate with others and increase our brand